.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 yarn.lock
+tags
```

Comment:
Not clear what this is for?

Reply:
this is a ctags file, which is part of some IDEs/toolchains like vim, emacs, etc.

Reply:
That should be in your own global gitignore, not here.